ts41/get.d.ts

```diff
@@ -2,7 +2,7 @@ import {Split} from './utilities';
 import {StringDigit} from '../source/utilities';
 
 /**
-Like @see {@link Get} but receives an array of strings as a path parameter.
+Like @see Get but receives an array of strings as a path parameter.
```

Comment:
Per the linked page, you cannot have `@see` in the middle of a sentence.

Reply:
🤦 sorry.